spec/models/movement_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+describe Movement, type: :model do
+  describe "validations" do
+    it { should validate_presence_of :movement_name }
+    it { should validate_presence_of :image }
+    it {should validate_presence_of :video }
+    it {should validate_presence_of :description }
+  end
+
+  describe "relationships" do
+    it { should belong_to :athlete }
+    it { should have_many :movement_goals}
+    it { should have_many :progress_movements}
+  end
```

Comment:
Add

it { should have_many(:goals).through(:movement_goals) }
it { should have_many(:progresses).through(:progress_movements) }